make.js

```diff
@@ -307,11 +309,22 @@ target.dist = function() {
     GENERIC_DIR + 'build/pdf.js',
     GENERIC_DIR + 'build/pdf.worker.js',
     SINGLE_FILE_DIR + 'build/pdf.combined.js',
+    MINIFIED_DIR + 'build/pdf.min.js',
+    MINIFIED_DIR + 'build/pdf.worker.min.js',
+    MINIFIED_DIR + 'build/pdf.combined.min.js'
   ], DIST_DIR + 'build/');
 
   mkdir('-p', DIST_DIR + 'web/');
   cp('-R', [
     COMPONENTS_DIR + '*',
+    GENERIC_DIR + 'web/viewer.js',
```

Comment:
The viewer is this repository is not intended to be used as-is by third-party solutions. Third-party viewers should always be different from the default viewer. Most people want a simple viewer, which they can achieve by using https://github.com/mozilla/pdf.js/tree/master/examples/components *or* by using parts of the default viewer. The default viewer is in this repository for two reasons: open-source development for the viewer in Firefox (and the Chrome extension) and third-party viewers might reuse parts of it.

It is really out of scope for this PR to include the viewer though.